website/pages/chapters/ridge.js

```diff
@@ -0,0 +1,845 @@
+              unregularized predictor. Remember, without regularization, it's
+              possible for the coefficients to be large if the model is
+              overfitting.
+            </p>
+          </TYU.Explanation>
+        </TYU>
+
+        <p>
+          Now consider another extreme when we make <IM math={`\\lambda`} />{" "}
+          large.
+        </p>
+
+        <TYU>
+          <TYU.Header>
+            What happens if make <IM math={`\\lambda`} /> so large that it
+            approaches <IM math={`\\infty`} />? What can you say about the
+            learned predictor?
+          </TYU.Header>
+          <TYU.Explanation>
+            <p>
+              This case is a bit trickier to think about
+              <MarginNote>
+                We are a bit hand-wavy in our argument since{" "}
+                <IM math={`\\infty`} /> isn't actually a number, but we use it
+                to think about a number getting arbitrarily large. You can make
+                all of the arguments below more formal by introducing limits,
+                but we avoid that for simplicity.
+              </MarginNote>
+              . Remember, we are trying to minimize this quality metric, so our
+              goal is to find a setting for{" "}
+              <IM math={`w = [w_0, w_1, ..., w_D]`} /> that minimizes{" "}
+              <IM math={`RSS(w) + \\lambda||w||_2^2`} />. In this extreme case,
+              where <IM math={`\\lambda`} /> approaches <IM math={`\\infty`} />,
+              our predictor will incur a high cost for any non-zero coefficient.
+              Therefore, the only valid predictor is one where all coefficients
+              are 0. What follows is an argument for why this is the case.
+            </p>
+
+            <p>
+              Consider using a predictor with some <IM math={`w_j \\neq 0`} />.
+              In that case <IM math={`||w||_2^2 > 0`} /> since{" "}
+              <IM math={`w_j`} /> will contribute some magnitude to the sum.
+              When we are considering this{" "}
+              <IM math={`\\lambda \\rightarrow \\infty`} /> case, the sum{" "}
+              <IM math={`RSS(w) + \\lambda||w||_2^2`} /> will also tend towards{" "}
+              <IM math={`\\infty`} />. We use the intuition that any non-zero
+              number times a number that goes to infinity, will also go to
+              infinity. This means{" "}
+              <IM math={`\\lambda||x||_2^2 \\rightarrow \\infty`} /> since{" "}
+              <IM math={`||x||_2^2 > 0`} />. Even if that magnitude is small,
+              you can blow up that penalty by choosing a super large{" "}
+              <IM math={`\\lambda`} />.
+            </p>
+
+            <p>
+              Consider a different case where every coefficient{" "}
+              <IM math={`j`} />, <IM math={`w_j = 0`} />. We would say then that
+              the whole vector <IM math={`w = 0`} /> as a notational
```

Comment:
consider boldfacing the 0 or adding a vector arrow over the 0?

Reply:
I'll boldface to avoid the arrow notation. Not that it's bad, but it's just another notational thing I want to avoid if strictly necessary (and I mention the `w=0` is a notational convenience).